CONTRIBUTING.md

```diff
@@ -0,0 +1,12 @@
+# Contributing
+
+## How to do submit your first contribution
+
+1. Fork this repository to create a copy of this repository in your account
+2. Clone the forked repository to your local machine
+3. Create a branch with a name that describes your intended change
+4. Make the necessary changes and commit those changes
+   * Add your name to the `Contributors.md` file
+5. Push changes to your fork on GitHub
+6. Go to your repository on GitHub and click the `Compare & Pull Request` button
+7. Request that your change be merged.
```

Comment:
Perhaps add click on 'Create Pull Request" button

Reply:
Good point, that would clear things up